src/components/AuthComponent/Auth.tsx

```diff
@@ -10,15 +10,17 @@ export interface IAuthProps {
 const AuthComponent = ({
   className = 'a6y-react-auth',
 }: IAuthProps): JSX.Element => {
-  const [currentForm] = useState('sign-in')
+  const [currentForm, setCurrentForm] = useState('/sign-in')
   const getAuthForm = (): JSX.Element => {
     switch (currentForm) {
-      case 'sign-up':
-        return <SignUp />
-      case 'forgot-password':
-        return <ForgotPassword />
+      case '/sign-up':
```

Comment:
NIT: I think that you could stay with `sign-in` instead of `/sign-in`, but it is also ok, so you don't have to change anything here ;)